### PR TITLE
paper for JOSS

### PR DESCRIPTION
**Pull request by aghaynes**
This is a paper for the Journal of Open Source Software. Don't worry about how it looks, JOSS needs markdown for their system... Note that citations are currently as references to the bib file (e.g. @bland2009) - they will be parsed and replaced by latex (or whatever system JOSS uses to make the PDFs; if you clone my repo to your computer and knit the file with RStudio, you will also see the more-final references).
To see the file in a closer state to complete, you can open it from my repo, [here](https://github.com/aghaynes/presize/blob/master/paper/paper.md). 
You probably don't need to look at the bib file, unless you notice a typo or something in a reference.

### CODE REVIEW

**Comment by aghaynes** [comment]
To comment, go to "Files changed"...

image

...scroll down to the text and find where your comment belongs. Clicking beside the line number will open a comment box.

image

File: paper/paper.bib [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  volume = {56},
+  number = {2},
+  pages = {85-89},
+  year = {2002},
+  doi = {10.1198/000313002317572736}
+  }
+
+  @article{ba1986,
+  author = {J M Bland and D G Altman},
+  title = {Statistical methods for assessing agreement between two methods of clinical measurement },
```
Comment (ostalder) [inline paper/paper.bib:25]: space at the end in the title.

File: paper/paper.bib [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  journal = {Lancet},
+  volume = {i(8476)},
+  pages = {307-310},
+  doi = {10.1016/S0140-6736(86)90837-8}
+}
+
+@ARTICLE{bland2007,
+  author = {J M Bland and D G Altman},
+  title = {Agreement between methods of measurement with multiple observations
+	per individual},
```
Comment (ostalder) [inline paper/paper.bib:36]: Why line return? Maybe it does not matter.

File: paper/paper.bib (hunk 1 above)
Comment (ostalder) [inline paper/paper.bib:25]: The titles are not consistent, capital letters.

File: paper/paper.bib [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  year = {2007},
+  volume = {17},
+  pages = {571-582},
+  number = {4},
+  doi = {10.1080/10543400701329422},
+  url = {https://www.ncbi.nlm.nih.gov/pubmed/17613642}
+}
+
+@article{bland2009,
+  author = {Bland, John Martin},
```
Comment (ostalder) [inline paper/paper.bib:47]: Names not consistant with the other ones. Only one letter for first name.

File: paper/paper.bib [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+  year = {2015},
+  journal = {Statistical Methods in Medical Research},
+  volume = {24},
+  number = {2},
+  pages = {224-254},
+  doi = {10.1177/0962280211415469}
+}
+
+@article{hm1982,
+  author = {J A Hanley and B J {McNeil}},
```
Comment (ostalder) [inline paper/paper.bib:104]: {} around McNeil, correct?

File: paper/paper.bib [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+  doi = {10.1214/ss/1009213286},
+  volume = {16},
+  number = {2},
+  pages = {101-117},
+  doi = {10.1214/ss/1009213286}
+}
+
+@article{bw2000,
+  author = {D G Bonnett and T A Wright},
+  year = 2000,
```
Comment (ostalder) [inline paper/paper.bib:84]: Does it change anything if 2000 instead of {2000}?

File: paper/paper.md [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+Risk difference | `prec_riskdiff` | Newcombe [@newcombe1998], Miettinen-Nurminen [@mn1985], Agresti-Caffo [@ac2000], Wald
+**Relative differences** | |
+Odds ratio | `prec_or` | Gart, Wolff, independence smoothed logit [see @fll2015]
+Risk ratio | `prec_riskratio` | Koopman [@koopman1984], Katz [@kbap1978]
+Rate ratio | `prec_rateratio` | Rothman [@rg2018]
+**Correlation measures** | |
+Correlation coefficient | `prec_cor` | Pearson, Kendall, Spearman [see @bw2000]
+Intraclass correlation | `prec_icc` | @bonnett2002
+Limit of agreement | `prec_lim_agree` | @ba1986
+Cohens kappa | `prec_kappa` | @rd2012
```
Comment (ostalder) [inline paper/paper.md:82]: Shouldn't we name it Cohen's kappa?

File: paper/paper.md [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+We developed a comprehensive and easy-to-use software package for precision-based 
+sample size calculation. As far as we know, presize is the first package that comprises 
+the most common summary measures used in estimation-based clinical research.
+A limitation of the package is that it does not allow calculating the probability 
+of a CI, i.e. the probability that a future confidence interval 
+will have at least the desired precision. The functions currently return the average 
+CI width. In practice, 50% of trials will yield narrower CIs and 
+50% will yield wider CIs due to sampling variation. Providing a method to specify 
+the coverage probability is one possible avenue 
+for further development
```
Comment (ostalder) [inline paper/paper.md:178]: . at the end of the sentence.

File: paper/paper.bib (hunk 2 above)
Comment (aghaynes) [reply paper/paper.bib:36]: that reference isnt actually used... will remove

File: paper/paper.bib (hunk 1 above)
Comment (aghaynes) [reply paper/paper.bib:25]: bibtex will handle that

File: paper/paper.bib (hunk 5 above)
Comment (aghaynes) [reply paper/paper.bib:84]: nope

File: paper/paper.md (hunk 6 above)
Comment (aghaynes) [reply paper/paper.md:82]: changed

File: paper/paper.md (hunk 7 above)
Comment (aghaynes) [reply paper/paper.md:178]: thanks

File: paper/paper.bib (hunk 3 above)
Comment (aghaynes) [reply paper/paper.bib:47]: i think bibtex will handle that too, but i'll change it anyway

**Comment by aghaynes** [comment]
Thanks @ostalder. @a-lenz and @alimacher, over to you...

File: paper/paper.md [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+`presize` is available on [CRAN](https://CRAN.R-project.org/package=presize) or [GitHub](https://github.com/CTU-Bern/presize) and can be loaded into the R session using
+
+```r
+# installation:
+# install.packages("presize") # CRAN
+# remotes::install_github('ctu-bern/presize') # development version on GitHub
+library(presize)
+```
+Suppose we want to estimate the proportion of hospital admissions with diabetes. 
+Diabetes has a prevalence of approximately 10% (@diab) and assume that we have a
```
Comment (alimacher) [inline paper/paper.md:112]: Consider revising: Diabetes has a prevalence of approximately 10% (@diab). We assume a slightly higher proportion of diabetics...

File: paper/paper.md [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+Rate ratio | `prec_rateratio` | Rothman [@rg2018]
+**Correlation measures** | |
+Correlation coefficient | `prec_cor` | Pearson, Kendall, Spearman [see @bw2000]
+Intraclass correlation | `prec_icc` | @bonnett2002
+Limit of agreement | `prec_lim_agree` | @ba1986
+Cohen's kappa | `prec_kappa` | @rd2012
+**Diagnostic measures** | |
+Sensitivity | `prec_sens` | As per `prec_prop`
+Specificity | `prec_spec` | As per `prec_prop`
+Area under the curve | `prec_auc` | @hm1982
```
Comment (alimacher) [inline paper/paper.md:86]: Add positive and negative likelihood ratios.

File: paper/paper.md (hunk 8 above)
Comment (aghaynes) [reply paper/paper.md:112]: done

File: paper/paper.md (hunk 9 above)
Comment (aghaynes) [reply paper/paper.md:86]: thanks

**Comment by aghaynes** [comment]
Thanks @alimacher. @a-lenz, your turn :)

**Comment by aghaynes** [comment]
no response from @a-lenz. Assume he agrees (as discussed with @alimacher).

**Pull request merged**